ui/src/pages/backoffice/Document/DocumentDetails/components/DocumentMetadata/DocumentSystemInfo.js

```diff
@@ -31,18 +32,27 @@ export class DocumentSystemInfo extends Component {
         value: toShortDateTime(document.updated),
       },
     ];
-    if (!isEmpty(document.metadata.created_by)) {
-      rows.push({
-        name: 'Created by',
-        value: `${document.metadata.created_by.type}: ${document.metadata.created_by.value}`,
-      });
-    }
-    if (!isEmpty(document.metadata.updated_by)) {
-      rows.push({
-        name: 'Last updated by',
-        value: `${document.metadata.updated_by.type}: ${document.metadata.updated_by.value}`,
-      });
-    }
+
```

Comment:
Maybe, in the future, we want to have the user id clickable to jump to the user profile page and see the name/email of who created/updated it :) For another time ;)